backend/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "target": "ES6",
+    "module": "NodeNext",
+    "strict": true,
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "noEmitOnError": true,
+    "strictNullChecks": true,
+    "allowJs": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true,
+    "outDir": "./dist",
+    "rootDir": "./src"
```

Comment:
FYI, checkout the baseURL option if you want to try cleaning up the import paths a bit https://www.typescriptlang.org/tsconfig/#baseUrl

Reply:
I like it! I'll add it to the tsconfig.

Reply:
On second thought, I think it's best to leave the paths as they are in case we're using a bundler that doesn't know about the mappings. It might lead to complications down the line.